packages/discord.js/src/structures/Message.js

```diff
@@ -1110,9 +1170,14 @@ class Message extends Base {
    * @returns {boolean}
    */
   equals(message, rawData) {
-    if (!message) return false;
+    if (!message) {
+      return false;
+    }
+
     const embedUpdate = !message.author && !message.attachments;
-    if (embedUpdate) return this.id === message.id && this.embeds.length === message.embeds.length;
+    if (embedUpdate) {
```

Comment:
Move the condition directly into the if() now, as `embedUpdate` isn't used elsewhere.